live-examples/js-examples/string/string-substring.html

```diff
@@ -0,0 +1,10 @@
+<pre>
+<code id="static-js">var str = 'Mozilla';
+
+console.log(str.substring(1, 2));
+// expected output "oz"
```

Comment:
This actually returns "o" not "oz". Sorry, should have caught this last time round.

Also, sorry to be super-nitpicky, but the format is:

```
// expected output: thing
```

Note the colon. Sorry to be so nitpicky, but it's worth being exact about this I think.

Reply:
I should have caught "oz" too. I fixed the code so it produces the intended result and also fixed the comments.